Hazel/src/Hazel/EntryPoint.h

```diff
@@ -7,9 +7,9 @@ extern Hazel::Application* Hazel::CreateApplication();
 int main(int argc, char** argv)
 {
 	Hazel::Log::Init();
-	HZ_CORE_WARN("Initialized Log!");
+	HZ_LOG_WARN("Initialized Log!");
```

Comment:
I am not entirely sure if it is intentional or not but these log messages get logged on the APP side instead of the Hazel Engine side because it is compiled in the Sandbox project.

Reply:
Yeah, I think the intent was good, but the macros only get resolved at compile time, same goes for templated classes, those could potentialy have the same issue... I would say leave them seperated into two...